src/main/java/com/amazon/aocagent/models/Context.java

```diff
@@ -38,18 +38,24 @@
   private String ecsLaunchType;
 
   /** ECS deployment mode. Eg, SIDECAR or DaemonSet. */
-  private String deploymentMode;
+  private String ecsDeploymentMode;
 
+  /** ECS cluster name. */
   private String clusterName;
```

Comment:
can we also add "ecs" to all the related parameters? like taskrolearn, clustername?

Reply:
yes,